common/dynamicconfig/constants.go

```diff
@@ -2528,6 +2528,16 @@ If the service configures with archival feature enabled, update worker.historySc
 		`How long to sleep within a local activity before pushing to workflow level sleep (don't make this
 close to or more than the workflow task timeout)`,
 	)
+	WorkerEnableDeployment = NewNamespaceBoolSetting(
+		"worker.enableDeployment",
+		false,
+		`WorkerEnableDeploymentGroup controls whether to start the worker for deployment and deployment-name workflows`,
+	)
+	DeploymentNamespaceStartWorkflowRPS = NewNamespaceFloatSetting(
```

Comment:
I'm not sure we need this? These are started/signalled as a result of frontend api calls, so there's already a rate limit there. We probably don't need another one